flow/Makefile

```diff
@@ -207,6 +207,8 @@ ifneq ($(USE_LSORACLE),)
   YOSYS_FLAGS ?= -m $(LSORACLE_PLUGIN)
 endif
 
+YOSYS_FLAGS += -v 3
```

Comment:
Adding -v 3 reduces verbosity?  That's non-intuitive...

Reply:
I looked in the pdf documentation and did not find anything useful. `yosys -h` returns:
```
    -v <level>
        print log headers up to level <level> to the console. (this
        implies -q for everything except the 'End of script.' message.)
```
Thus, `-v 3` will print:
```
14.2.4. Analyzing design hierarchy
```

While `-v 2` would print:
```
14.2. Executing AST frontend in derive mode using pre-parsed AST for module `\aes_cipher_top'.
```

Did not find a finer knob than this.